zeus/run/dataloader.py

```diff
@@ -1011,17 +1011,17 @@ def __next__(self):
                     self.train_epoch_time.append(time_consumed)
                     # Record the energy consumption for each GPU.
                     for index in range(self.world_size):
-                        self.train_epoch_energy[index][
-                            self.epoch_num - 1
-                        ] = energy_consumed[index]
+                        self.train_epoch_energy[index][self.epoch_num - 1] = (
+                            energy_consumed[index]
+                        )
```

Comment:
Did `black` make this change? I'd rather not change unrelated stuff. Can you see if you can revert this change? Although if `black` insists then we'll have to comply and change.

Reply:
yes, black made this change. I'll revert it and try to ignore it

Reply:
reverted and passed lint